### ISSUE TITLE
[BUG] installation instructions not working on cloud9

### ISSUE THREAD

**Issue by smazanek** [aws-cloudformation/cloudformation-guard#72]
**Describe the bug**
I try to setup cloudformation guard on a fresh cloud9 IDE according to the current installation instructions for Linux from https://github.com/aws-cloudformation/cloudformation-guard
i.e.:
wget https://github.com/aws-cloudformation/cloudformation-guard/releases/download/VERSION/cfn-guard-linux-VERSION.tar.gz
tar -xvf cfn-guard-linux-1.0.0.tar.gz
cd ./cfn-guard-linux
./cfn-guard 

When I try to run ./cfn-guard I receive following error:
./cfn-guard: /lib64/libc.so.6: version `GLIBC_2.18' not found (required by ./cfn-guard)

**Comment by smazanek** [comment]
What is the best way on cloud9 to fulfill the prerequisites?

**Comment by rajddas** [comment]
Getting the same error!

**Comment by jacobleonwoffenden** [comment]
I've built `cfn-guard` using CentOS 7, you can download it from here https://github.com/maglevlabs/cloudformation-guard/releases/tag/0.1.1

**Comment by rajddas** [comment]
@jacobleonwoffenden  Thanks Man. It did work for me. But still waiting for AWS's official fix.

**Comment by jotompki** [comment]
Thanks we will take a look at this and #71 for distributing a CentOS compatible binary. In the meantime if you have rust installed try running `cargo install cfn-guard` which will place the binary in the `~/.cargo/bin/` directory.

**Comment by smazanek** [comment]
Thanks, @jotompki works for me!

**Comment by rajddas** [comment]
Yeah it worked for me too. I was trying to include this part in CodeBuild Install phase. But the cargo install is taking a bit time to install. 
Waiting for the permanent fix.

**Comment by rajddas** [comment]
Is there any update on this issue? Trying to get a working official tarball executable to be installed on a on-prem bamboo server.
We are not able to install rust due to security issues given by admin team. I am working on it, meanwhile it would be great if a working tar.gz is available.

**Comment by jotompki** [comment]
Centos7 binary is now included in the 1.0.0 release. In the future we will vend only this one, but until we have another release I wanted to unblock: 

https://github.com/aws-cloudformation/cloudformation-guard/releases/download/1.0.0/cfn-guard-centos-1.0.0.tar.gz
